__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index.js");
+const request = require("supertest");
+beforeEach(() => {
+  return seed(data).then(() => db.end);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("tests for GET /api/topics", () => {
+  test("should return 200 status code", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("should return correct result with expected properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(3);
+        body.forEach((topic) => {
+          expect(topic).toHaveProperty("slug");
+          expect(topic).toHaveProperty("description");
+          expect(typeof topic.slug).toBe("string");
+          expect(typeof topic.description).toBe("string");
+        });
+      });
+  });
+  test("should return 404 status code if path was wrong", () => {
```

Comment:
This test actually applies to all endpoints and should have it's own describe block please